site/components/src/test/java/uk/nhs/digital/intranet/provider/MicrosoftGraphAuthorizationProviderTest.java

```diff
@@ -137,4 +138,11 @@ public void throwsAuthorizationExceptionIfGraphApiThrowsHttpExceptionOnRefreshAc
         headers.setContentType(MediaType.APPLICATION_FORM_URLENCODED);
         return new HttpEntity<>(map, headers);
     }
+
+    private boolean timeWithin5Seconds(AccessToken accessToken) {
```

Comment:
Added more flexibility to time check ensuring that the token calculated is within 5 seconds of the correct time rather than exact second which was causing unit tests to fail intermittently.